### PR TITLE
Fixes #23 Added support autogenerate Slice

### LINKED ISSUE #23: Add support to define GoSlice in generate config
## Expected Behavior
- By means of definition in the configuration file, define which are the SLICE

## Real behavior
- It must be opened manually in the library.

### PR DESCRIPTION
**Pull request by m4yk3ldev**
Fixes #23 

Changes:
- Autogenerate GoSlice to  parameter is []typedata
- Configure in file extern to typedata in slice not added

Does this change need to mentioned in CHANGELOG.md?

No

### CODE REVIEW

File: src/cmd/cgogen.go [hunk 1]
```diff
@@ -84,14 +84,16 @@ var (
...
 
-var arrayTypes = map[string]string{
-	"PubKey":    "cipher",
-	"SHA256":    "cipher",
-	"Sig":       "cipher",
-	"SecKey":    "cipher",
-	"Ripemd160": "cipher",
-	"UxArray":   "coin",
-}
+//var arrayTypes = map[string]string{
```
Comment (olemis) [inline src/cmd/cgogen.go]: remove it if not needed

**Review by olemis — CHANGES_REQUESTED** [review]

File: src/cmd/cgogen.go (hunk 1 above)
Comment (m4yk3ldev) [reply src/cmd/cgogen.go]: Yes, because these specific types are sent by the configuration file, since in all the libc they are not equal.

Comment (olemis) [reply src/cmd/cgogen.go]: The code is commented out , there is no need to keep it . Please remove it

Comment (m4yk3ldev) [reply src/cmd/cgogen.go]: ok

**Review by olemis — APPROVED** [review]

**Pull request merged**